js/index.js

```diff
@@ -64,7 +64,14 @@ class Libraries {
 const title = document.querySelector('#text_title');
 const author = document.querySelector('#text_author');
 const add = document.querySelector('#add');
-
+// const for GUI
+const item1 = document.querySelector('#item1');
+const item2 = document.querySelector('#item2');
+const item3 = document.querySelector('#item3');
+const bigContainer = document.querySelector('.big-container');
+const anchors = document.querySelectorAll('nav ul a');
+const timeTag = document.querySelector('span');
+timeTag.innerHTML = `${new Date().toLocaleString()}`;
```

Comment:
- [ ] The current date is displayed below the navigation bar :heavy_check_mark:, however, This requirement is not fully implemented, the date and time is present but you have to use [Luxon](https://moment.github.io/luxon/api-docs/index.html) as it was required in project requirement, try that out so that it will help you learn more about different libraries.